pom.xml

```diff
@@ -56,7 +56,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
-				<version>3.2.4</version>
+				<version>3.3.0-SNAPSHOT</version>
```

Comment:
this should just be `3.3.0`, no `-SNAPSHOT`

Reply:
Thanks for the heads-up! :smile: That version seems to have been released about a month ago :slightly_smiling_face: